webapp/src/components/AccordionLayout.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { Typography } from "@mui/material";
+import { fireEvent, screen } from "@testing-library/react";
+import AccordionLayout from "components/AccordionLayout";
+import { renderWithTheme } from "mocks/utils";
+
+type Props = {
+  name: string;
+  description: string;
+  link: string;
+  children: React.ReactNode;
+};
+
+const renderAccordionLayout = (accordionLayout: Props) =>
+  renderWithTheme(<AccordionLayout {...accordionLayout} />);
+
+describe("AccordionLayout", () => {
+  it("should display name, description and description component on the accordian layout", () => {
+    renderAccordionLayout({
+      name: "Project Config",
+      description:
+        "Contains mandatory fields that specify the dataset to load in Azimuth",
+      link: "reference/configuration/project/",
+      children: <Typography>Project Config Section</Typography>,
+    });
```

Comment:
I don't see value in `renderAccordionLayout()`, as well as duplicating the definition of `Props` here. React components are already meant to be rendered, no need to wrap them in functions. Here is what I'd do. Note the "accordian" misspelled with an "a".
```suggestion
describe("AccordionLayout", () => {
  it("should display name, description and description component on the accordion layout", () => {
    renderWithTheme(
      <AccordionLayout
        name="Project Config"
        description="Contains mandatory fields that specify the dataset to load in Azimuth"
        link="reference/configuration/project/"
      >
        <Typography>Project Config Section</Typography>
      </AccordionLayout>
    );
```

Reply:
Earlier I planned to write different test cases for title and description, so I declared this common function. But it makes sense to me now to declare it inside, as there is one test case inside the describe.